src/Razor/test/Microsoft.AspNetCore.Razor.VSCode.FunctionalTest/tests/TestUtil.ts

```diff
@@ -136,7 +136,7 @@ export async function waitForProjectReady(directory: string) {
 export async function waitForProjectsConfigured() {
     await pollUntil(() => {
         const files = glob.sync(`**/${projectConfigFile}`, { cwd: testAppsRoot});
-        return files.length === 16;
+        return files.length === 17;
```

Comment:
Kind of like how we find how many project.razor.json files are in the directory can we just find how many csproj's are in the directory instead of hardcoding the number?